packages/apidom-ls/src/config/openapi/openapi3_1/lint/jsonSchemaDialect--type.ts

```diff
@@ -0,0 +1,16 @@
+import ApilintCodes from '../../../codes';
```

Comment:
This must be implemented as `json-schema-dialect--format-uri.ts`. 

Notice the fine name convention and the `--format-uri`. The documentation says following about this field: `This MUST be in the form of a URI.`

Reply:
will fix